act-rules-format.md

```diff
@@ -170,16 +170,18 @@ Each [=accessibility requirement=] in the mapping <em class="rfc2119">must</em>
 4. the conformance level associated with the accessibility requirement, if one exists.
 
 
-### Outcome Mapping ### {#outcome-mapping}
+### Mapping Conformance Requirements ### {#mapping-conformance-requirements}
 
-For each accessibility requirement in the mapping, an ACT Rule <em class="rfc2119">must</em> indicate what the [=outcomes=] of the rule mean for satisfying that accessibility requirement for that [=test subject=]. When one or more of the outcomes for a test target is `failed`, the accessibility requirements are <dfn>not satisfied</dfn> for the test subject. When all of the outcomes are `passed` or `inapplicable`, the accessibility requirements could be <dfn>satisfied</dfn>, or <dfn>further testing is needed</dfn>. Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
+An ACT Rule <em class="rfc2119">must</em> indicate what the [=outcomes=] of the rule mean for satisfying an accessibility requirement for that [=test subject=]. A conformance requirement mapped in a rule is an accessibility requirement that is <dfn>not satisfied</dfn> when all of the outcomes for a test target in failed test cases is `failed`. When all of the outcomes for passed test cases or inapplicable test cases are `passed` or `inapplicable`, respectively, the mapped conformance requirement could be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn>. Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
```

Comment:
```suggestion
An ACT Rule <em class="rfc2119">must</em> indicate what the [=outcomes=] of the rule mean for satisfying an accessibility requirement for that [=test subject=]. A conformance requirement mapped in a rule is an accessibility requirement that is <dfn>not satisfied</dfn> when one or more outcomes for a test target is `failed`. When all of the outcomes are `passed` or `inapplicable`, the mapped conformance requirement could be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn>, but cannot be <dfn>not satisfied</dfn>. Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
```

Reply:
Above suggestion reverts back to the original text and adds that passed outcomes cannot be not satisfied to move stricter requirement Contrast (enhanced) out of the conformance requirement mapping of the rule for Contrast (minimum).